kernel/kernel-defaults/src/test/scala/io/delta/kernel/defaults/TestUtils.scala

```diff
@@ -127,4 +124,27 @@ trait TestUtils {
       TimeZone.setDefault(currentDefault)
     }
   }
+
+  /** All simple data type used in parameterized tests where type is one of the test dimensions. */
```

Comment:
It used to be called `PRIMITIVE_TYPES`, but changed it because of [this](https://github.com/delta-io/delta/pull/1997#discussion_r1320017946) review comment.